React Native/Icons.md

```diff
@@ -0,0 +1,44 @@
+There are two methods on how we use and implement icons. The first one is by adding 
```

Comment:
What it means `custom svg icon files`?
Maybe this should be more explained, since as far as I am aware just importing svg icons doesn't work. You have to either use `react-native-svg` library or custom transformer to use svg icons as is or "custom".

Reply:
I can change this to something like:

The first one is by adding custom svg icon files to the project and using a library called react-native-svg ...

In the next paragraph is a reference to another handbook section (which is in review currently) that explains in detail how to use this method.


What do you think about this change?